tests/default_installation.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { test, expect } from '@playwright/test';
```

Comment:
should be default-installation.spec.ts
Connected with the different formats `-` or `_` what do you think about https://progress.opensuse.org/issues/134228#note-11 we should have those folders ci/ and e2e/ to distinguish properly, could you file ticket in case you don't plan it here? it will help IMO.